src/App.jsx

```diff
@@ -1,6 +1,19 @@
-import './App.css'
+import './App.css';
+import ArticlesList from './components/ArticlesList';
+import Header from './components/Header';
+import { Route, Routes } from "react-router-dom";
+import Nav from './components/Nav';
```

Comment:
Nav's not being used in here, so line 5 should be removed. Otherwise, this is a great start to routing!